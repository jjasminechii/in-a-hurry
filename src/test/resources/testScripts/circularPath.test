# Find a path with Dijkstra's algorithm when
# the path is circular.

CreateGraph simpleGraph

AddNode simpleGraph A
AddNode simpleGraph B
AddNode simpleGraph C
AddNode simpleGraph D

AddEdge simpleGraph A B 10.0
AddEdge simpleGraph B C 1.0
AddEdge simpleGraph C A 1.0
AddEdge simpleGraph A D 1.0

FindPath simpleGraph A D