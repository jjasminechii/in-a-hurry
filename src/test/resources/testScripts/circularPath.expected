# Find a path with Dijkstra's algorithm when
# the path is circular.

created graph simpleGraph

added node A to simpleGraph
added node B to simpleGraph
added node C to simpleGraph
added node D to simpleGraph

added edge 10.000 from A to B in simpleGraph
added edge 1.000 from B to C in simpleGraph
added edge 1.000 from C to A in simpleGraph
added edge 1.000 from A to D in simpleGraph

path from A to D:
A to D with weight 1.000
total cost: 1.000